.gitignore

```diff
@@ -4,7 +4,7 @@ npm-debug.log
 .vscode
 .tmp
 
-dist
```

Comment:
is there significance to this?

Reply:
I had `lighthouse-viewer/dist` left after this PR from an earlier build and it'll just stay there since it's gitignored. This will kind of alert that it's unnecessary now, but otherwise not much significance :)